README.md

```diff
@@ -3121,6 +3132,8 @@ end
 
 Use [Doorkeeper](https://github.com/doorkeeper-gem/doorkeeper), [warden-oauth2](https://github.com/opperator/warden-oauth2) or [rack-oauth2](https://github.com/nov/rack-oauth2) for OAuth2 support.
 
+You can access the controller context with the `#context` method inside any auth middleware inherited from `Grape::Middlware::Auth::Base`.
```

Comment:
Can we clarify here why you'd want to use the `#context`. For example
```
you can access the controller params, headers, and helpers through the context with the `#context` method inside any auth middleware inherited from `Grape::Middlware::Auth::Base`.
```

Reply:
(Unless I'm mistaken and you can't access the helpers)

Reply:
Yes, the helpers are accessible :) 
I'll improve the wording

Reply:
Done